http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpHailBatchDAO.scala

```diff
@@ -0,0 +1,38 @@
+package org.broadinstitute.dsde.workbench.leonardo.dao
+
+import cats.effect.Async
+import cats.mtl.Ask
+import cats.syntax.all._
+import org.broadinstitute.dsde.workbench.leonardo.AppContext
+import org.broadinstitute.dsde.workbench.openTelemetry.OpenTelemetryMetrics
+import org.http4s._
+import org.http4s.client.Client
+import org.http4s.client.dsl.Http4sClientDsl
+import org.http4s.headers.Authorization
+
+class HttpHailBatchDAO[F[_]](httpClient: Client[F])(implicit
+  F: Async[F],
+  metrics: OpenTelemetryMetrics[F]
+) extends HailBatchDAO[F]
+    with Http4sClientDsl[F] {
+
+  override def getStatus(baseUri: Uri, authHeader: Authorization)(implicit
```

Comment:
Could you leave a comment explainng the difference between the regular status, and the driver status? I went through the design doc, but it still is a bit unclear to me